homeassistant/components/sma_manager/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "domain": "sma_manager",
+    "name": "SMA Manager",
+    "codeowners": ["DEADSEC-SECURITY"],
+    "dependencies": [],
+    "documentation": "",
```

Comment:
Not OK, please add a documentation link (https://home-assistant.io/integrations/sma_manager/)

Reply:
How am I supposed to get docs url, if the docs page was not merged yet by HA. Its kinda of a circular dependency, they won't merge if this is not merged but this won't merge if docs don't exist? @emontnemery

Reply:
The docs URL is guaranteed to match the integration domain, so you can safely assume it will be https://home-assistant.io/integrations/sma_manager/